src/acuvity/response/evaluator.py

```diff
@@ -0,0 +1,76 @@
+from typing import Optional
+
+from ..guard.config import Guard
+from ..models.scanresponse import Scanresponse
+from ..utils.logger import get_default_logger
+from .parser import ResponseParser
+from .result import GuardMatch, ResponseMatch
+
+logger = get_default_logger()
+
+class ResponseEvaluator:
+    """
+    Handles pure check evaluation without considering actions.
```

Comment:
what does this mean?

Reply:
I think, it was the 1set version with actions. changed.